spec/requests/api/editor/editor_can_see_company_statistics_spec.rb

```diff
@@ -3,9 +3,10 @@
   let!(:editor_headers) { editor.create_new_auth_token }
   let!(:journalist) { create(:user, role: 'journalist') }
   let!(:journalist_headers) { journalist.create_new_auth_token }
-  let!(:published_articles1) { create(:article, created_at: Time.zone.now) }
-  let!(:published_articles2) { create(:article, created_at: Time.zone.now - 100_000)  }
-  let!(:published_articles3) { create(:article, created_at: Time.zone.now - 200_000)  }
+  let!(:published_article1) { create(:article, created_at: Time.zone.now) }
+  let!(:published_article2) { create(:article, created_at: Time.zone.now - 100_000)  }
+  let!(:published_article3) { create(:article, created_at: Time.zone.now - 200_000)  }
```

Comment:
what dose 100_000 stuff do?

Reply:
It's just a separator apparently, done by Solar :p